helix-core/src/lib.rs

```diff
@@ -48,15 +51,26 @@ pub fn find_root(root: Option<&str>) -> Option<std::path::PathBuf> {
 
 #[cfg(not(embed_runtime))]
 pub fn runtime_dir() -> std::path::PathBuf {
```

Comment:
Can you make this private now? We always want to use the cached variable

Reply:
Done.